tests/scene/test_code_edit.h

```diff
@@ -2831,6 +2831,16 @@ TEST_CASE("[SceneTree][CodeEdit] indent") {
 		CHECK(code_edit->has_selection());
 		CHECK(code_edit->get_selection_origin_column() == 2);
 		CHECK(code_edit->get_caret_column() == 3);
+
+		// Multiline blocks.
+		code_edit->set_text("'''\n    test\n        test\n'''");
+		code_edit->select(1, 0, 1, 9);
```

Comment:
```suggestion
		code_edit->select(1, 0, 1, 8);
```

This was being clamped, we can set it directly to column 8 so that the `get_caret_column()` later is more clear.